types/raspi-i2c/index.d.ts

```diff
@@ -0,0 +1,35 @@
+// Type definitions for raspi-i2c 6.1
+// Project: https://github.com/nebrius/raspi-i2c
+// Definitions by: Bryan Hughes <https://github.com/nebrius>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.1
+
+/// <reference types="node" />
+import { Peripheral } from 'raspi-peripheral';
+export type IReadCallback = (err: null | Error | string, data: null | Buffer | number) => void;
```

Comment:
Nit: Don't begin type names with `I` unnecessarily

Reply:
Good catch, I missed that one, thanks. Fixed.